### ISSUE TITLE
[BUG] AnimatePresence doesn't unmount "exited" children 

### ISSUE THREAD

**Issue by vovaguguiev** [framer/motion#178]
**Describe the bug**
AnimatePresence doesn't unmount "exited" children 

**Expected**
After the exit animation is finished, the exited child unmounts

**Actual**
All the previously exited children stay in DOM

**Version**
1.2.0

**Code to reproduce**

```jsx
import * as React from "react"
import { Frame } from "framer"
import { motion, AnimatePresence } from "framer-motion"

export function Billboard({
    width,
    height,
    url,
    background,
    backgroundSize,
    transitionType,
    damping,
    stiffness,
    duration,
    easing,
}) {
    const animation = {
        initial: { opacity: 0 },
        active: { opacity: 1 },
        exit: { opacity: 0 },
    }

    const transition =
        transitionType === "spring"
            ? { type: "spring", damping, stiffness }
            : { type: "tween", duration, ease: easing }

    return (
        <Frame size="100%" overflow="hidden" background={background}>
            <AnimatePresence initial={false}>
                <motion.div
                    key={url}
                    initial={animation.initial}
                    animate={animation.active}
                    exit={animation.exit}
                    transition={transition}
                    style={{
                        position: "absolute",
                        width,
                        height,
                        backgroundImage: `url(${url})`,
                        backgroundSize,
                        backgroundRepeat: "no-repeat",
                    }}
                />
            </AnimatePresence>
        </Frame>
    )
}
```

**Comment by InventingWithMonster** [comment]
Closed: As this is a Framer X integration issue we'll take a look internally.

**Comment by vovaguguiev** [comment]
It’s somehow fixed in 1.2.2 btw
